pkg/scheduler/internal/cache/cache.go

```diff
@@ -379,7 +379,7 @@ func (cache *cacheImpl) AssumePod(pod *v1.Pod) error {
 	cache.mu.Lock()
 	defer cache.mu.Unlock()
 	if _, ok := cache.podStates[key]; ok {
-		return fmt.Errorf("pod %v is in the cache, so can't be assumed", key)
+		return fmt.Errorf("pod %v is in the cache, so can't be assumed", klog.KObj(pod))
```

Comment:
UID rather than namespace/name was chosen as key to avoid tricky issue when a pod with the same namespace/name get recreated in a short period. So in order not to lose the UID info in log, I'd suggest appending the pod's namespace/name info instead of replacing it.

Reply:
`cacheImpl` is an internal struct, it doesn't care about the pod name and namespace, so return key seems enough. But we can add the log at a higher level via `klog.XXX()`.

Reply:
@Huang-Wei Sounds good. I'll add pod namespace/name info to the log:  `pod key(pod info)`. 

@kerthcet , I think we wanted to keep the info. in the context. Actually, in the cache.go log, there are quite a few msgs with high level info. such as node, etc.

Reply:
> UID rather than namespace/name was chosen as key to avoid tricky issue when a pod with the same namespace/name get recreated in a short period. So in order not to lose the UID info in log, I'd suggest appending the pod's namespace/name info instead of replacing it.

Sounds good. I'll add pod namespace/name info to the log:  `podKey(podInfo)`.

Reply:
> `cacheImpl` is an internal struct, it doesn't care about the pod name and namespace, so return key seems enough. But we can add the log at a higher level via `klog.XXX()`.

I think we wanted to keep the info. in the context. Actually, in the cache.go log, there are quite a few msgs with high level info. such as node info, etc. As Wei suggested, we keep pod key(UID) and add additional pod info.